src/components/tool/guide/Config.js

```diff
@@ -50,27 +50,42 @@ const lyraConfigOptions = (config) => {
     }
 }
 
-export function ConfigGroup({title, description, type, selected, inputProps}) {
+export function ConfigGroup({title, description, type, selected, inputProps, showAlert = false, alertType = "info", alertMsg = ""}) {
     return (
         <Box>
-            <Flex align='center'>
-                <Box w='120px'>
-                    {selected ?
-                        <CheckCircleIcon color="green.500"/> :
-                        <Tooltip label="Pick an option"><WarningIcon color="orange.500"/></Tooltip>
+            <Flex direction="column" width="100%">
+                <Flex direction="row" align='center'>
+                    <Box w='120px'>
+                        {selected ?
+                            <CheckCircleIcon color="green.500"/> :
+                            <Tooltip label="Pick an option"><WarningIcon color="orange.500"/></Tooltip>
+                        }
+                        <Tooltip label={description} placement="top" hasArrow>
+                            <Heading size='xs' mb="0" display="inline" ml="2">{title}</Heading>
+                        </Tooltip>
+                    </Box>
+                    <Square size="10px"/>
+                    {
+                        type === "picker" &&
+                        <InputPicker {...inputProps}/>
                     }
-                    <Tooltip label={description} placement="top" hasArrow>
-                        <Heading size='xs' mb="0" display="inline" ml="2">{title}</Heading>
-                    </Tooltip>
-                </Box>
-                <Square size="10px"/>
-                {
-                    type === "picker" &&
-                    <InputPicker {...inputProps}/>
-                }
+                    {
+                        type === "slider" &&
+                        <InputSlider {...inputProps}/>
+                    }
+                </Flex>
                 {
-                    type === "slider" &&
-                    <InputSlider {...inputProps}/>
+                    showAlert === true &&
+                    <Flex direction="row" align='center'>
+                        <Box minWidth='120px'/>
+                        <Square size="10px"></Square>
+                        <Box w='100%'>
+                            <Alert status={alertType}>
```

Comment:
What do you think about making the alert dismissible by the user (ie. with a `x` or `Acknowledge` button)? I think it makes sense because if the user understands what they're doing, they should be able to dismiss the warning visually as well as mentally. But interested in your thoughts.